lib/common.js

```diff
@@ -125,23 +135,11 @@ module.exports.createTemporaryFilename = fileType => {
  */
 
 function hashArray(arr) {
```

Comment:
breaking change, FWIW (previously it returned a promise - if somebody does `.then` instead if `await`, it'll crash)

Reply:
can make the function `async` to retain compatibility, if you care about it

Reply:
oh yea, i thought about it in terms of "it still works the same with or without the `await` keyword" but that of course doesnt take into account the use of .then.

Reply:
I think either a breaking change or a second method `hashArraySync` or some such

Reply:
went with breaking change